doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -117,7 +84,7 @@ Overview
 
    # hack hack
    git add my_new_file
-   git commit -am 'ENH: some message'
+   git commit -a
```

Comment:
``git status`` before commit makes sense here, I disagree with removing ``-a`` - it's what you want 9 out of 10 times.  ``git add`` is still needed for a new file (which is rare), but not for changes to existing files.